wp-content/mu-plugins/pantheon-mu-plugin/pantheon.php

```diff
@@ -37,4 +37,10 @@
 		 */
 		define( 'FS_METHOD', 'direct' );
 	}
+
+    // When developing a WordPress Multisite locally, ensure that this constant is set.
+    // This will set the Multisite variable in all Pantheon environments.
+    if (getenv('FRAMEWORK') == 'wordpress_network' && !defined('WP_ALLOW_MULTISITE')) {
```

Comment:
This file is added to the repository by updatinator, which means on a WordPress update, this will get overwritten by https://github.com/pantheon-systems/pantheon-mu-plugin. This change should go [there](https://github.com/pantheon-systems/pantheon-mu-plugin/blob/main/pantheon.php) if that's where it's going to go, and it will be updated here on a new WP release.